src/plugins/input-configurator/public/webcomponents/orov-input-configurator.html

```diff
@@ -7,683 +7,642 @@
-                    this.$.savenewpresetmodal.open();
+                openUpdatePresetToast: function() {
+                    var self = this;
+
+                    self.candidateBinding = undefined;
+                    self.$.updatedPresetToast.open();
                 },
 
-                onSaveNewPreset: function() {
+                //Reset the candidateBinding on keyboard modal closed
+                onKeyboardModalClosed : function() {
                     var self = this;
-                    if (self.eventEmitter) {
-                        self.eventEmitter.emit('plugin.inputConfigurator.saveNewPreset', self.newPresetName);
-                        self.set('saveNewPreset', false);
-                    }
+                    
                 },
-                onSavePreset: function() {
+                
+                //Reset the candidateBinding on gamepad modal closed
+                onGamepadModalClosed : function() {
                     var self = this;
-                    if (self.eventEmitter) {
-                        self.eventEmitter.emit('plugin.inputConfigurator.savePreset', self.presets[self.selectedPreset]);
-                        self.set('saveNewPreset', false);
-                    }
+                    
                 },
+                
+                //Function to achieve the sticky table headers with pure CSS.
+                tableContainerScroll: function() {
+                    var self = this;
 
-                getClass: function(duplicates, duplicateClass, nonDuplicateClass) {
-                    if (duplicates) {
-                        return duplicateClass;
-                    }
-                    return nonDuplicateClass ? nonDuplicateClass : '';
+                    /*CSS way to do sticky headers*/
+                    var translate = "translate(0," + self.$.tableContainer.scrollTop + "px)";
+                    self.$.stickyHeader.style.transform = translate;
                 },
 
-                onReset: function() {
-                    if (this.selectedItem) {
-                        // $(this.$.keyboard).val(this.selectedItem.defaults.keyboard);
-                        this.set('selectedItem.bindings.keyboard', this.selectedItem.defaults.keyboard == undefined ? null : this.selectedItem.defaults.keyboard);
-                        // $(this.$.gamepad).val(this.selectedItem.defaults.gamepad);
-                        this.set('selectedItem.bindings.gamepad', this.selectedItem.defaults.gamepad == undefined ? null : this.selectedItem.defaults.gamepad);
-                        this.keyboardDuplicates = [];
-                        this.gamepadDuplicates = [];
-                    }
+                //Handles clearing an existing keyboard command
+                handleClearKeyboardBinding: function() {
+                    var self = this;
+
+                    //Clear the input from our presets
+                    console.log("Clearing:", self.selectedItem);
```

Comment:
Be sure to replace the console.log messages with the debug utilities to eliminate the console messages.